udata/templates/dataset/card.html

```diff
@@ -3,11 +3,11 @@
 {% from theme('macros/certified.html') import badge_if_certified with context %}
 <div class="card dataset-card">
     {% if dataset.organization %}
-        <a class="card-logo" href="{{ dataset_url }}">
+        <div class="card-logo">
```

Comment:
Why drop the link to the org? Confusing?

Reply:
I found that confusing indeed: the link targets the Dataset whereas the `alt` attribute and the image signpost the org.